src/app/base/components/node/OverviewCard/ControllerStatusCard/ControllerStatusCard.test.tsx

```diff
@@ -167,12 +178,17 @@ it("renders correct version info for an unknown install type", async () => {
   await userEvent.hover(
     screen.getByRole("button", { name: Labels.VersionDetails })
   );
-  expect(
-    within(screen.getByRole("tooltip")).getByLabelText(Labels.Version)
-  ).toHaveTextContent("Version: Unknown (less than 2.3.0)");
-  expect(
-    within(screen.getByRole("tooltip")).getByLabelText(Labels.Origin)
-  ).toHaveTextContent("Origin: nowhere");
+  await vi.waitFor(() => {
+    expect(
+      within(screen.getByRole("tooltip")).getByLabelText(Labels.Version)
+    ).toHaveTextContent("Version: Unknown (less than 2.3.0)");
+  });
+
+  await vi.waitFor(() => {
+    expect(
+      within(screen.getByRole("tooltip")).getByLabelText(Labels.Origin)
+    ).toHaveTextContent("Origin: nowhere");
+  });
```

Comment:
```suggestion
  await waitFor(() => {
    expect(
      within(screen.getByRole("tooltip")).getByLabelText(Labels.Origin)
    ).toHaveTextContent("Origin: nowhere");
  });
```

Reply:
Does this cause the test to fail? I'd suggest to stick to a single convention (and we use `waitFor` from testing-library instead).

Reply:
Just found an example of this - if you go to `src/app/base/components/DebounceSearchBox/DebounceSearchBox.test.tsx` and replace `vi.waitFor` with `waitFor` (testing lib), the test will fail

IMO, if we were to choose one or the other, I would go with `vi.waitFor`. Maybe we should update the export in our test utils to export this instead of the one from testing lib

Reply:
Let's do that! https://github.com/canonical/maas-ui/pull/5478